modules/gdnative/godot/godot_vector2.h

```diff
@@ -61,34 +61,26 @@ godot_real GDAPI godot_vector2_length_squared(const godot_vector2 *p_v);
 godot_real GDAPI godot_vector2_distance_to(const godot_vector2 *p_a, const godot_vector2 *p_b);
 godot_real GDAPI godot_vector2_distance_squared_to(const godot_vector2 *p_a, const godot_vector2 *p_b);
 
-// @Incomplete
-/*
- * missing:
- *
- *	angle_to
- *	angle_to_point
- *	dot
- *	cross_vector
- *	cross_scalar
- *	project
- *	plane_project
- *	clamped
- *	linear_interpolate
- *	cubic_interpolate
- *	cubic_interpolate_soft
- *	slide
- *	reflect
- *	angle
- *	abs
- *	rotated
- *	tangent
- *	floor
- *	snapped
- *	aspect
- *
- *
- *	to_string
- */
+void GDAPI godot_vector2_abs(godot_vector2 *p_dest, const godot_vector2 *p_src);
+godot_real GDAPI godot_vector2_angle(const godot_vector2 *p_src);
+godot_real GDAPI godot_vector2_angle_to(const godot_vector2 *p_src, const godot_vector2 *p_to);
+godot_real GDAPI godot_vector2_angle_to_point(const godot_vector2 *p_src, const godot_vector2 *p_to);
+void GDAPI godot_vector2_clamped(godot_vector2 *p_dest, const godot_vector2 *p_src, godot_real length);
+void GDAPI godot_vector2_cubic_interpolate(godot_vector2 *p_dest, const godot_vector2 *p_src,
+										   const godot_vector2 *p_b, const godot_vector2 *p_pre_a,
+										   const godot_vector2 *p_post_b, godot_real t);
```

Comment:
I'm not sure if travis agrees with me, but the way you split longer function names isn't really pretty 😄 
I don't have anything against aligning function arguments, but I usually *align* stuff with spaces, so it looks the same everywhere. Using tabs makes it look inconsistent in different editors.

I personally don't care if it would be one long name or indented with spaces, but tabs are a bit .... weird 😄